challenge_2/python/alexbotello/main.py

```diff
@@ -0,0 +1,6 @@
+if __name__ == '__main__':
+    N = int(input()) # How many items in the list?
+    array = [int(input()) for _ in range(N)] # Enter ints
+
+    print("\nThe single number is ", end='')
+    print(*[i for i in array if array.count(i) == 1], sep=' or ')
```

Comment:
Thanks for letting me know. I'm just learning list comprehensions trying to use them more often in my code.
Would using the sorted function in the comprehension remedy this?